nncf/openvino/graph/model_transformer.py

```diff
@@ -466,3 +468,41 @@ def _apply_bias_insertion_transformations(
                 node_output_source_port.replace_source_output(add_node.output(0))
 
         return model
+
+    @staticmethod
+    def _apply_multiply_insertion_transformations(
+        model: ov.Model, transformations: List[OVMultiplyInsertionCommand]
+    ) -> ov.Model:
+        """
+        Inserts Multiply with provided value for corresponding layer.
+
+        :param transformations: List of the smooth insertion transformations.
+        :returns: Transformed model with Multiply nodes.
+        """
```

Comment:
Minor comment: we can unify this insertion function in future to make it inserting any kind of operations, for example bias add or smooth quant multiply. I believe next time we will need to insert something we should think about unification